servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -899,14 +900,14 @@ void RenderForwardMobile::_render_scene(RenderDataRD *p_render_data, const Color
 			// regular forward for now
 			Vector<Color> c;
 			{
-				Color cc = clear_color.srgb_to_linear();
+				Color cc = clear_color.srgb_to_linear().darkened(inverse_luminance_multiplier);
```

Comment:
This should just be a multiply operation instead of using the `darkened` operation

Reply:
Does this apply to both lines using `darkened`, or just this one?

Reply:
Both